Sources/StreamChat/Workers/Background/AttachmentQueueUploader.swift

```diff
@@ -131,13 +133,12 @@ class AttachmentQueueUploader: Worker {
             // Update attachment local state.
             attachmentDTO.localState = newState
 
-            if let uploadedAttachment = uploadedAttachment {
-                var attachment = uploadedAttachment.attachment
-                let remoteUrl = uploadedAttachment.remoteURL
-
-                self?.updateRemoteUrl(of: &attachment, with: remoteUrl)
-
-                attachmentDTO.data = attachment.payload
+            if var uploadedAttachment = uploadedAttachment {
+                self?.updateRemoteUrl(of: &uploadedAttachment)
+                if let processedAttachment = self?.attachmentPostProcessor?.process(uploadedAttachment: uploadedAttachment) {
+                    uploadedAttachment = processedAttachment
```

Comment:
Would be nice to have a test that processes an attachment.

Reply:
The reason I didn't add a test here is that IMO it is a bit waste of time. It will be too much setup to prepare this test, to test 1 obvious thing. But I can add it if you think it is still worth it 👍

Reply:
Up to you to decide, based on the effort for setup. The quality gate for coverage also fails this way.

Reply:
Test added ✅